src/key_listener.c

```diff
@@ -0,0 +1,25 @@
+#include "key_listener.h"
+
+t_bool	is_key_arrow_left(const char *key)
+{
+	return (key[0] == ESCAPE_CHAR
+		&& key[1] == BRACKET_LEFT
+		&& key[2] == CHARACTER_D);
+}
+
+t_bool	is_key_new_line(const char *key)
+{
+	return (key[0] == NEW_LINE);
+}
+
+t_bool	is_key_printable(const char *key)
+{
+	if (ft_isprint(key[0]))
+		return (TRUE);
+	return (FALSE);
+}
+
+t_bool	is_key_backspace(const char *key)
+{
+	return (key[0] == DELETE);
```

Comment:
With names such as DELETE, it might be good practice to either only define them in a .c file or make it something like TC_ESCAPE (termcaps_escape) or something.